docs/tutorials/wiki/src/authorization/tutorial/security.py

```diff
@@ -1,5 +1,16 @@
-USERS = {'editor':'editor',
-          'viewer':'viewer'}
+import bcrypt
+
+
+def hash_password(pw):
+    return bcrypt.hashpw(pw.encode('utf-8'), bcrypt.gensalt())
```

Comment:
I would like wiki and wiki2 to have consistent implementations, whichever is preferred.